js/views/aliases.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * ownCloud - Mail
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ * @copyright Tahaa Karim 2016
+ */
+
+define(function(require) {
+	'use strict';
+
+	var Marionette = require('marionette');
+	var AliasesListView = require('views/aliases-list');
+	var Backbone = require('backbone');
+	var Radio = require('radio');
+
+	return Marionette.CollectionView.extend({
+		collection: null,
+		childView: AliasesListView,
+		currentAccount: null,
+		initialize: function(options) {
+			this.currentAccount = options.currentAccount;
+			this.collection = new Backbone.Collection(this.currentAccount.get('aliases'));
```

Comment:
we should pass the alias collection from the outside, which should be a property of the account. Then Marionette will do the automatic view updating for you